src/alr/alr-commands-search.adb

```diff
@@ -90,7 +132,8 @@ package body Alr.Commands.Search is
          Tab.Append ("external");
          Tab.Append (Alire.Index.Crate (Name).TTY_Description);
          Tab.Append (Ext.Image);
-      end List_Undetected;
+         Tab.Append (Match_Locations.Flatten (", "));
```

Comment:
These match locations could be a Set instead of a Vector, to avoid for example:
```
$ alr search hello
...
ppag    2.1.1    First steps with Gnoga       Executable, Executable, Executable, Executable, Executable, Executable, Executable, Executable, Executable, Executable, Executable, Executable, Executable, Executable
```